src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+
+$month-length: 31;
+$week-day: (mon:7, tue:6, wed:5, thu:4, fri:3, sat:2, sun:1);
+
+$block-width: 100px;
+$container-width: 710px;
```

Comment:
it works fine for these sizes of gaps and blocks, but if you change them, it won't work correctly